pkg/storegateway/storepb/custom.go

```diff
@@ -6,6 +6,9 @@
 package storepb
 
 import (
+	"slices"
+	strings "strings"
```

Comment:
```suggestion
	"strings"
```

Reply:
Strange that the automation would make a prefix automatically :/ Thanks.